scripts/schemas/board-schema.yaml

```diff
@@ -0,0 +1,255 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2023, Nordic Semiconductor ASA
+
+# JSON Schema for board metadata YAML files
+# When possible, constraints and validation rules should be modeled directly in this file.
+
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://zephyrproject.org/schemas/zephyr/board-schema.json"
```

Comment:
@pdgendt oops, this was left unnoticed :) I am tempted to make the ID `https://zephyrproject.org/schemas/zephyr/board` -- thoughts?

Reply:
Yes, sounds good!